### PR TITLE
Document RDF Generation for Repo Managers.

### PR DESCRIPTION
**Pull request by rosiel**
**GitHub Issue**: None directly but I blame #1286 

# What does this Pull Request do?

Documents:

- how the JSON-LD module works, its api and its single config option
- How the JSON-LD module works with the Drupal RDF module
- What the Drupal RDF module does
- Why we use the ?_format=jsonld business - i.e. how the JSONLD module works with the RESTful Web Services module
- what mappings look like (refreshed to current data) and how to edit them (slightly reduced but clarified, IMO)

# What's new?

DOCS!

# How should this be tested?

Read https://github.com/rosiel/islandora-1/blob/documenting-rdf-generation/docs/islandora/rdf-mapping.md 

(or, better, clone this branch and use mkdocs for prettier `!!!note` formatting.)

# Additional Notes:

This was exhausting. But bring on the corrections.

# Interested parties
📢 @Islandora-CLAW/committers

### CODE REVIEW

File: docs/islandora/rdf-mapping.md [hunk 1]
```diff
@@ -1,39 +1,132 @@
...
+- The JSON-LD module has a configuration option that can cause the `?_format=jsonld` to be part of, or not part of, the URIs of Drupal objects. On an out-of-the-box claw-playbook, this string is stripped, but by default on a fresh install of the jsonld module, it is not.
+
+
+## RDF Mappings
+In an out-of-the-box claw-playbook, the RDF mappings that exist were loaded from config files, and correspond to the rdf.mapping.[...].yml files located in:
+
+- `[drupal modules directory]/islandora/modules/islandora_core_feature/config/install/` (media and taxonomy terms)
+- `[drupal modules directory]/islandora_defaults/config/install/` (`repository_item` and the `islandora_access` vocabulary)
+- `[drupal web root]/core/profiles/standard/config/install/` (articles, pages, comments, and tags).
+
```
Comment (seth-shaw-unlv) [inline docs/islandora/rdf-mapping.md:47]: Also, `[drupal modules directory]/controlled_access_terms/modules/controlled_access_terms_defaults/config/install/` (for controlled access terms vocabularies).

File: docs/islandora/rdf-mapping.md [hunk 2]
```diff
@@ -1,39 +1,132 @@
...
+### How to edit an RDF Mapping
+
+Once loaded, RDF mappings can be customized for the needs of a particular site through Drupal's Configuration Synchronization UI at `admin/config/development/configuration`. They can be exported, modified, and re-imported one-at-a-time by choosing the "Single Item" option on the Export/Import tabs.  You can also create new RDF mappings (e.g. for a custom content type) and load them through this interface, by copying an existing mapping and changing the appropriate values.
+
+!!! note "Possible future UI for RDF Mappings"
+    A custom module `rdfui` exists, and is installed-but-not-enabled on boxes provisioned by the claw-playbook. We don't use it because it is very rudimentary and limited to the schema.org vocabulary. There is a project underway to develop a [UI](https://github.com/Islandora-CLAW/CLAW/issues/647) to support RDF mappings to any ontology.
```
Comment (seth-shaw-unlv) [inline docs/islandora/rdf-mapping.md]: Despite the presence of a ticket and some initial conversations, I don't know that the UI development is 'underway'. Atleast, I'm not aware of anyone actively working on it.

File: docs/islandora/rdf-mapping.md [hunk 3]
```diff
@@ -1,39 +1,132 @@
...
+
+- A number of namespaces such as `ldp`, `ebucore`, `pcdm`, are `premis` are registered in `islandora.module` using `hook_rdf_namespaces()`. To register your own namespaces, you will need to create a custom module that implements that hook.
+- If you import a configuration that uses a namespace that is not registered, bad things will happen silently.
+
+
+### Structure of an RDF Mapping
+Below is an example of an RDF mapping as a .yml (YAML) file. It is the current version of the RDF mapping of the Repository Item (`islandora_object`) bundle, provided by `islandora_defaults` and exportable as `rdf.mapping.node.islandora_object.yml`).
+
+- The top level key `types` specifies the `rdf:type` of the resource or content model. `field_model`, a required field of Islandora objects, also gets mapped to `rdf:type` through an arcane back-end process.
```
Comment (seth-shaw-unlv) [inline docs/islandora/rdf-mapping.md:64]: controlled_access_terms _used_ to have the ability to magically update the type via a configured field (e.g. corporate_body's field_type). But that appears to have been lost in the shuffle...

File: docs/islandora/rdf-mapping.md (hunk 2 above)
Comment (rosiel) [reply docs/islandora/rdf-mapping.md]: was just copying from a previous version!

File: docs/islandora/rdf-mapping.md (hunk 1 above)
Comment (rosiel) [reply docs/islandora/rdf-mapping.md:47]: thanks!

File: docs/islandora/rdf-mapping.md (hunk 3 above)
Comment (rosiel) [reply docs/islandora/rdf-mapping.md:64]: Danny referred to this in a comment in the issue: https://github.com/Islandora-CLAW/CLAW/issues/1286#issuecomment-536669582

**Comment by rosiel** [comment]
@seth-shaw-unlv I have finally figured out why you called it controlled_access_terms. 😸

**Review by dannylamb — APPROVED** [review]
:+1: Let's pull this in so it can be iterated on in the sprint.

**Pull request merged**